Description: disable unaligned FP accesses emulation on armhf, as Ubuntu does
 not have definitions for `struct user_vfp` and `struct user_vfp_exc`.
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/js/src/wasm/WasmSignalHandlers.cpp
+++ b/js/src/wasm/WasmSignalHandlers.cpp
@@ -243,7 +243,6 @@ using mozilla::DebugOnly;
 // emulation here.
 
 #if defined(__linux__) && defined(__arm__)
-#  define WASM_EMULATE_ARM_UNALIGNED_FP_ACCESS
 #endif
 
 #ifdef WASM_EMULATE_ARM_UNALIGNED_FP_ACCESS

